api/v1/storagecluster_types.go

```diff
@@ -456,18 +456,33 @@ type EncryptionSpec struct {
 	Enable bool `json:"enable,omitempty"`
 	// +optional
 	ClusterWide bool `json:"clusterWide,omitempty"`
+	// Configure the default rbd encrypted storage class
```

Comment:
can we remove the word default as we are not creating defaults with this one? use `RBD`

Reply:
Done